openfl/federated/plan/plan.py

```diff
@@ -421,7 +421,7 @@ def get_tensor_pipe(self):
 
     def get_straggler_handling_policy(self):
         """Get straggler handling policy."""
-        template = "openfl.component.straggler_handling_functions.CutoffTimeBasedStragglerHandling"
+        template = "openfl.component.aggregator.straggler_handling.CutoffPolicy"
```

Comment:
Is this default? What is the default behavior?

Reply:
Yes. In case no policy is specified in plan then straggler handling policy is disabled.